addons/web/static/tests/core/hotkeys/hotkey_service_tests.js

```diff
@@ -672,3 +672,18 @@ QUnit.test("protects editable elements: can bypassEditableProtection", async (as
         "the callback still gets called even if triggered from an editable"
     );
 });
+
+QUnit.test("ignore numpad keys", async (assert) => {
+    assert.expect(1);
+
+    const key = '1';
+
+    env.services.hotkey.add(`alt+${key}`, () => assert.step(key));
+    await nextTick();
+
+    let keydown = new KeyboardEvent("keydown", { key, code: "Numpad1", altKey: true });
+    window.dispatchEvent(keydown);
+    await nextTick();
+
+    assert.verifySteps([]);
```

Comment:
add another keydown event asserting that it works when event.code is something else than NumpadX, like DigitX (which is the code format for keys from the numeric row)

Reply:
good job btw ;-)